beta/src/components/MDX/Sandpack/CustomPreset.tsx

```diff
@@ -73,15 +73,16 @@ export function CustomPreset({
             <SandpackCodeEditor
               customStyle={{
                 height: getHeight(),
-                maxHeight: isExpanded ? '' : 406,
+                maxHeight: isExpanded ? '' : '40vh',
+                minHeight: isExpanded ? '' : 'min-h-40vh xl:min-h-0',
               }}
               showLineNumbers
               showInlineErrors
               showTabs={false}
             />
             <Preview
               isExpanded={isExpanded}
-              className="order-last xl:order-2"
+              className="order-last min-h-40vh xl:order-2 xl:min-h-0"
```

Comment:
```suggestion
              className="order-last min-h-[40vh] xl:order-2 xl:min-h-0"
```
May be we could do this instead of config level changes?

Reply:
What’s that? Some new syntax?

Reply:
Arbitrary values in tailwind https://tailwindcss.com/docs/min-height#arbitrary-values